packages/samples/react/src/components/tabs/behavior-select-manuel.tsx

```diff
@@ -0,0 +1,60 @@
+import type { FC } from 'react';
+import React from 'react';
+
+import { KolTabs } from '@public-ui/react';
+import { SampleDescription } from '../SampleDescription';
+
+const tabs = [
+	{
+		_icons: 'codicon codicon-pie-chart',
+		_label: 'First tab',
+		_on: {
+			onSelect: (event: Event) => {
```

Comment:
(Vorschlag:) Persönlich würde ich das Beispiel weiter reduzieren um den Fokus ganz auf die Behavior-Prop zu setzen. Ein Beispiel ohne Icons und Callbacks wäre in dem Sinne für mich ausreichend.

Was aber interessant sein könnte: Ein zweites Beispiel auf der selben Seite, dass das Standard-Behavior im direkten Vergleich zeigt.

Reply:
Wenn ich die React Sample App wie eine Art Stroybook sehe, dann wäre es hier eine eigene Story, da ich ja ein neues Feature eingebaut habe.

Reply:
@Makko74 Dieses Beispiel so umstricken, dass es ein Tabs mit Manuell und ein Tabs mit Automatic zeigt. Du kannst ja gerne Tabs mit Icons bei beiden Tabs machen. Aber nur das Behavior unterschiedlich machen. Beispiele jeweil betiteln.